backend/api-server/src/seeding/seeding.service.ts

```
@@ -28,155 +29,179 @@ export class SeedingService implements OnApplicationBootstrap {
-      const targetVenue = venues[Math.floor(Math.random() * venues.length)];
+    try {
+      await this.initVenuesAndBlocks();
+
+      // ID 매핑을 위한 메모리 로드
+      const venues = await this.venuesRepository.find();
+      const blocks = await this.blocksRepository.find({ relations: ['venue'] });
+
+      const venueMap = new Map<string, number>();
+      venues.forEach((v) => venueMap.set(v.venueName, v.id));
+
+      const blockMap = new Map<string, number>(); // Key: "VenueName:BlockName" -> Value: BlockId
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)